components/command/package.go

```diff
@@ -47,18 +47,23 @@ func (m *GenericPackageManager) Ensure(packageRef string, opts ...pulumi.Resourc
 
 		opts = append(opts, utils.PulumiDependsOn(updateDB))
 	}
-	installCmd := fmt.Sprintf("%s %s", m.installCmd, packageRef)
-	cmd, err := m.runner.Command(
-		m.namer.ResourceName("install-"+packageRef, utils.StrHash(installCmd)),
-		&Args{
-			Create:      pulumi.String(installCmd),
-			Environment: m.env,
-			Sudo:        true,
-		},
-		opts...)
+
+	cmdStr := fmt.Sprintf("%s %s", m.installCmd, packageRef)
+	cmdName := m.namer.ResourceName("install-"+packageRef, utils.StrHash(cmdStr))
+	cmdArgs := Args{
+		Create:      pulumi.String(cmdStr),
+		Environment: m.env,
+		Sudo:        true,
+	}
+	if customizer != nil {
+		cmdName, cmdArgs = customizer(cmdName, cmdArgs)
+	}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
	// Given a Customizer function, use it to transform the command
	if customize != nil {
		cmdName, cmdArgs = customize(cmdName, cmdArgs)
	}
```